components/dashboard/src/App.tsx

```diff
@@ -52,6 +53,7 @@ const InstallGitHubApp = React.lazy(() => import(/* webpackPrefetch: true */ './
 const FromReferrer = React.lazy(() => import(/* webpackPrefetch: true */ './FromReferrer'));
 const UserSearch = React.lazy(() => import(/* webpackPrefetch: true */ './admin/UserSearch'));
 const WorkspacesSearch = React.lazy(() => import(/* webpackPrefetch: true */ './admin/WorkspacesSearch'));
+const AdminSettings = React.lazy(() => import(/* webpackPrefetch: true */ './admin/Settings'));
```

Comment:
**issue(non-blocking)**: Just wanted to add a comment that I could not access the admin panel evern after trying to add a user as an admin using this documentation[[1](https://www.notion.so/gitpod/How-we-develop-e8ce7e562478458a9223eb9b797415b2#d0f67538075f4050ba7c2551c7160f3b)].

Reply:
I'm not sure about adding a user as admin. In my environment where my user is an admin by default, it shows ok. As you say, I think it's probably ok as there will always be the initial admin. We can always raise as a bugfix if a wider problem

Reply:
Yes, the first user always becomes an admin. That was more like an observation that I could not test this in the preview environment properly either because the script to make someone admin does not work or because we could improve our code review process.

Reply:
I've raised PR #7840 with a fix for leeway